src/graphics/program-lib/programs/standard.js

```diff
@@ -998,6 +1000,22 @@ var standard = {
         var useVsm = false;
         var usePerspZbufferShadow = false;
         var light;
+
+        var hasAreaLights = options.lights.some(function (light){
+            return light._shape !== LIGHTSHAPE_PUNCTUAL;
+        });
+
+        if (device._areaLightLutFormat === PIXELFORMAT_R8_G8_B8_A8) {
+            // use offset and scale for rgb8 format luts
+            code += "#define HAS_R8_G8_B8_A8_LUTS\n";
```

Comment:
and similarly here, not sure the HAS part is needed in HAS_R8_G8_B8_A8_LUTS
maybe: AREA_LUTS_R8_G8_B8_A8